pkg/csi/controller/server.go

```diff
@@ -145,11 +145,18 @@ func (c *Server) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		return nil, status.Errorf(codes.InvalidArgument, "unsupported filesystem type %v for volume %v", req.GetVolumeCapabilities()[0].GetMount().GetFsType(), name)
 	}
 
+	var volumeClaimID string
 	for key, value := range req.GetParameters() {
-		if key == string(directpvtypes.AccessTierLabelKey) {
+		switch key {
+		case string(directpvtypes.AccessTierLabelKey):
 			if _, err := directpvtypes.StringsToAccessTiers(value); err != nil {
 				return nil, status.Errorf(codes.InvalidArgument, "unknown access-tier %v for volume %v; %v", value, name, err)
 			}
+		case string(directpvtypes.VolumeClaimIDLabelKey):
+			if err := validateVolumeClaimID(value); err != nil {
```

Comment:
We could use regex (even it is little slower) instead of `uuid` package for precise check
```go
var volumeClaimIDRegex = regexp.MustCompile("^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$")
```
later here
```go
if !volumeClaimIDRegex.MatchString(value) {
    return nil, status.Errorf(codes.InvalidArgument, "invalid volume claim ID %v; ", value)
}
volumeClaimID = value;
```